partition/bootloader_uboot.go

```diff
@@ -165,6 +165,10 @@ func (u *uboot) GetBootVar(name string) (value string, err error) {
 	return getBootVar(name)
 }
 
+func (u *uboot) SetBootVar(name, value string) error {
+	return setBootVar(name, value)
```

Comment:
Thats a good point. I pushed a new branch feature/bootloader-set-bootvar2 that removes this double indirection.